PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -132,8 +144,8 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
     relative = cms.bool(False),
     rho_MiniIso = cms.InputTag("fixedGridRhoFastjetAll"),
     rho_PFIso = cms.InputTag("fixedGridRhoFastjetAll"),
-    EAFile_MiniIso = cms.FileInPath("RecoEgamma/ElectronIdentification/data/Fall17/effAreaElectrons_cone03_pfNeuHadronsAndPhotons_94X.txt"),
-    EAFile_PFIso = cms.FileInPath("RecoEgamma/ElectronIdentification/data/Fall17/effAreaElectrons_cone03_pfNeuHadronsAndPhotons_94X.txt"),
+    EAFile_MiniIso = cms.FileInPath("RecoEgamma/ElectronIdentification/data/Run3_Winter22/effAreaElectrons_cone03_pfNeuHadronsAndPhotons_122X.txt"),
```

Comment:
Don't we want to keep the Fall17 constants for Run2 samples?

Reply:
yes, @rgoldouz can you fix it please? It's better to not delete anything related to Run2 IDs, but just add things related to Run3 IDs.